frontend/src/components/collective/CollectiveExpenseItem.js

```diff
@@ -39,6 +39,13 @@ const CollectiveExpenseItem = ({
 
   let src, srcSet, href, receiptMessage, receiptHelpTip;
 
+  let submittedByName = user && user.name;
+
+  // show email only if the user can view receipt
+  if (canViewReceipt) {
```

Comment:
We don't need to decide that now. This is a one line change, let's get it out. Not setting a precedent here. Besides, the approval process should take care of that. When you `cover`, you trust that since it's approved, it's a legit expense.